resources/qml/Dialogs/AboutDialog.qml

```diff
@@ -181,6 +197,31 @@ UM.Dialog
         }
     }
 
+    AboutDialogVersionsList{
+        id: projectBuildInfoList
+
+    }
+
+    onShakeDetectedChanged:
+    {
+        if (!projectBuildInfoList.visible)
+        {
+            projectsList.visible= false;
+            projectBuildInfoList.visible = true;
+        }
+        else
+        {
+            projectsList.visible = true;
+            projectBuildInfoList.visible = false;
+        }
```

Comment:
You always flip the visibility on `onShakeDetectionChanged` right? why not write
```suggestion
        projectsList.visible = !projectsList.visible;
        projectBuildInfoList.visible = !projectBuildInfoList.visible;
```